build-packages.cmd

```diff
@@ -60,6 +60,15 @@ if NOT [!ERRORLEVEL!]==[0] (
   exit /b 1
 )
 
+rem Build the TestHost package
+set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.TestHost\Microsoft.NETCore.TestHost.builds" !allargs!
```

Comment:
I am not very sure why this script exists...but it had build commands for other package and I added testhost to it. @wtgodbe added this script.